datacapturing/src/main/java/de/cyface/datacapturing/model/GeoLocation.java

```diff
@@ -5,17 +5,22 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 import android.support.annotation.NonNull;
+import android.util.Log;
 
 import de.cyface.datacapturing.DataCapturingService;
 
 /**
  * A position captured by the {@link DataCapturingService}.
  *
  * @author Klemens Muthmann
- * @version 1.2.0
+ * @version 1.2.1
  * @since 1.0.0
  */
 public class GeoLocation implements Parcelable {
+    /**
+     * The tag used to identify Logcat output from objects of this class.
+     */
+    private static final String TAG = "de.cyface.capturing";
```

Comment:
The generic "capturing" TAG should be stored centrally not duplicated into each capturing class.

Reply:
Correct. However I would like to avoid such ground breaking changes in the current master. We should do that as part of the integration and release for next years version

Reply:
Sure ;-)